utils/website_crawler.py

```diff
@@ -0,0 +1,54 @@
+from HTMLParser import HTMLParser
+import urllib
+
+
+class WebsiteCrawler(HTMLParser):
+    def __init__(self, raw_html, website_url):
+        HTMLParser.__init__(self)
+        print("Crawling possible websites...")
+        self.website_set = set()
+        self.website_url = website_url
+        self.raw_html = raw_html
+
+        self.website_set.add(website_url)
+
+    def handle_starttag(self, tag, attrs):
+        if tag == "a":
+            for name, value in attrs:
+                if name == "href":
+                    if self.website_url not in value:
```

Comment:
wouldn't this incorrectly match if say we were searching jana.com and the href was twitter.com/jana.com?